Source/WebCore/rendering/RenderGrid.h

```diff
@@ -128,6 +128,7 @@ class RenderGrid final : public RenderBlock {
     LayoutUnit gridGap(GridTrackSizingDirection, std::optional<LayoutUnit> availableSize) const;
 
     LayoutUnit masonryContentSize() const;
+    Vector<LayoutRect> gridItemsLayoutRects();
```

Comment:
Let's make this a const function

Reply:
Discuss with Sammy.

This will require changing the orderiterator to support const better.